BuildTools/InstallSwiftDependencies.sh

```diff
@@ -11,7 +11,7 @@ then
     if [ "$SYSTEM_DISTRO" == "Debian" ]
     then
         sudo apt-get install build-essential pkg-config libssl-dev qt5-default libqt5x11extras5-dev libqt5webkit5-dev qtmultimedia5-dev qttools5-dev-tools qt5-image-formats-plugins libqt5svg5-dev libminiupnpc-dev libnatpmp-dev libhunspell-dev
-    elif [ "$SYSTEM_DISTRO" == "Ubuntu" ]
+    elif [ "$SYSTEM_DISTRO" == "Ubuntu" ] || [ "$SYSTEM_DISTRO" == "LinuxMint" ]
```

Comment:
Why not do both checks in the same test instead of running two commands, so an easy elif [ "$SYSTEM_DISTRO" == "Ubuntu" -o "$SYSTEM_DISTRO" == "LinuxMint" ] ?